docs/html/quickstart.rst

```diff
@@ -2,90 +2,126 @@
 Quickstart
 ==========
 
-First, :doc:`install pip <installing>`.
+pip is a command line tool for installing Python packages.
 
-Install a package from `PyPI`_:
+For a full tutorial on installing packages with pip, see the
+`Python Packaging User Guide`_
+
+For a more detailed exploration of pip's functionality see the pip
+:doc:`user guide <user_guide>`.
+
+Quick setup
+===========
+
+1. Check pip installation
+-------------------------
+Check if you already have pip installed by running:
+
+.. code-block:: console
+
+  pip --version
+
+See :doc:`installing pip <installing>` for more information, or to troubleshoot
+any problems.
+
+2. Check pip is up to date
+--------------------------
+
+To make sure you are running pip's latest version, run:
 
 .. tab:: Unix/macOS
 
    .. code-block:: console
 
-      $ python -m pip install SomePackage
-      [...]
-      Successfully installed SomePackage
+      python -m pip install -U pip
 
 .. tab:: Windows
 
    .. code-block:: console
 
-      C:\> py -m pip install SomePackage
-      [...]
-      Successfully installed SomePackage
+      py -m pip install -U pip
+
 
+3. Set up a virtual environment
+-------------------------------
 
-Install a package that's already been downloaded from `PyPI`_ or
-obtained from elsewhere. This is useful if the target machine does not have a
-network connection:
+Before using pip, we recommend you set up and use a virtual environment to
+isolate your project packages.
+
+See `Creating Virtual Environments`_ for more information.
+
+Common tasks
+============
+
+Install a package from `PyPI`_
+------------------------------
 
 .. tab:: Unix/macOS
 
    .. code-block:: console
 
-      $ python -m pip install SomePackage-1.0-py2.py3-none-any.whl
+      $ python -m pip install SomePackage
       [...]
       Successfully installed SomePackage
 
 .. tab:: Windows
 
    .. code-block:: console
 
-      C:\> py -m pip install SomePackage-1.0-py2.py3-none-any.whl
+      C:\> py -m pip install SomePackage
       [...]
       Successfully installed SomePackage
 
-Show what files were installed:
+Install a package from GitHub
+------------------------------
 
-.. tab:: Unix/macOS
+Pip can install packages from common version control systems (VCS), including
+GitHub.
 
-   .. code-block:: console
+For example, to install a specific commit from the Django project, run:
 
-      $ python -m pip show --files SomePackage
-      Name: SomePackage
-      Version: 1.0
-      Location: /my/env/lib/pythonx.x/site-packages
-      Files:
-      ../somepackage/__init__.py
-      [...]
+.. code-block:: console
 
-.. tab:: Windows
+  pip install git+https://github.com/django/django.git@45dfb3641aa4d9828a7c5448d11aa67c7cbd7966
 
-   .. code-block:: console
+See VCS support for more information.
```

Comment:
@pradyunsg I'd like to link to the VCS section in the user guide. How can I do this?